mof_array/pmf/process_mass_data.py

```diff
@@ -1,31 +1,44 @@
-"""This code imports mass adsorption data and "experimental" adsorption data
-from simulated output, for multiple MOFs and gas mixtures, and calculates the
-probability that specific gases are present in specific mole fractions
-(in each experimental case) based on the total mass adsorbed for each MOF.
-Additionally, the best MOF arrays for detecting each gas are reported,
-according to the highest information gain.
-"""
-import os
-import sys
-from math import isnan, log
+# Original Code by Jenna Gustafson
+# Modifications by Brian Day
+
+# Jenna:
+# ﻿This code imports mass adsorption data and 'experimental' adsorption data
+# from simulated output, for multiple MOFs and gas mixtures, and calculates the
+# probability that specific gases are present in specific mole fractions (in
+# each experimental case) based on the total mass adsorbed for each MOF.
+# Additionally, the best MOF arrays for detecting each gas are reported,
+# according to the highest information gain.
+
+# Brian:
+# Note on list comprehension: List comprehension will catch all instances, where
+# as simple for loops will only catch the last instance, unless all variables
+# are pre-allocated and appended to. Use list comprehension whenever possible.
+# In some instances of the code, using a mof twice would not be captured. Try to
+# fix this going forward.
+
+# --------------------------------------------------
+# ----- Import Python Packages ---------------------
+# --------------------------------------------------
 import csv
-from random import random
-from itertools import combinations
-from functools import reduce
-import operator
-
-import yaml
```

Comment:
follow pep8; keep groupings but ok to separate out imports / froms. https://www.python.org/dev/peps/pep-0008/#imports